entity/node/accessors/node_modules/iotAuthService.js

```diff
@@ -458,6 +465,8 @@ eventHandlers = {
 }
 */
 exports.sendSessionKeyReqViaTCP = function(options, sessionKeyRespHandler, eventHandlers, callbackParams) {
+    console.log("sendSessionKeyReqViaTCP was called");
```

Comment:
What's the purpose of these logs? Seems unnecessary.

Reply:
Oh, sorry, I'll take these out.